app/routes/goal_routes.py

```diff
@@ -1 +1,76 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, make_response, abort
+from ..db import db
+from .route_utilities import validate_model, update_dict_to_database, add_dict_to_database
+from app.models.goal import Goal
+from app.models.task import Task
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+@goals_bp.get("/<id>/tasks")
+def get_task_with_goal(id):
+    goal = validate_model(Goal, id)
+    return goal.to_dict(withRelationship=True)
```

Comment:
See my comment in the `Goal` model about how you can achieve the same functionality in `to_dict` without needing to have an argument `withRelationship`.
```suggestion
    return goal.to_dict()
```